lib/auth/usertoken.go

```
@@ -148,10 +148,11 @@ func (a *Server) CreateResetPasswordToken(ctx context.Context, req CreateUserTok
 		return nil, trace.BadParameter("invalid reset password token request type")
 	}
 
-	// ResetPassword doesn't error for non-existent users, so first check if the
-	// user exists.
-	if _, err = a.GetUser(ctx, req.Name, false /* withSecrets */); err != nil {
+	switch user, err := a.GetUser(ctx, req.Name, false /* withSecrets */); {
+	case err != nil:
 		return nil, trace.Wrap(err)
```

It didn't seem to me that important to hide the error here either.